docs/gather_data.rst

```diff
@@ -60,6 +60,91 @@ To gather data for the last N days or N hours, use ``--since``:
     $ # see data for the last 12 hours
     $ ./multivac/gather_job_data.py --since 12h
 
+Writing data to JSON file
+-------------------------
+
+To write data to JSON file, use ``--format json`` option. If you add `-t` option,
+script will write data about tests too:
```

Comment:
You've picked the right structure in the first sentence. Use it in the second as well.

task → solution. task2 → solution2

```suggestion
To write data to a JSON file, use the ``--format json`` option.
To gather data about tests as well, add the ``--tests`` or ``-t`` option:
```

Reply:
done